projects/go-tester/src/app/app.service.ts

```diff
@@ -5,18 +5,19 @@ import { GoTableConfig, GoTablePageConfig, GoTableSortConfig } from '../../../go
 
 // Only using this to emulate sorting for server side
 import { sortBy } from '../../../go-lib/src/lib/components/go-table/go-table-utils';
+import { Observable } from 'rxjs';
 
 @Injectable({
   providedIn: 'root'
 })
 
 export class AppService {
 
-  constructor (private http: HttpClient) { }
+  constructor(private http: HttpClient) { }
 
-  getMockData(params?: GoTableConfig) {
-    return this.http.get<any>("../assets/MOCK_DATA_1000.json").pipe(map(data => {
-      let formattedData = { totalCount: 0, results: [] };
+  getMockData(params?: GoTableConfig): Observable<any> {
+    return this.http.get<any>('../assets/MOCK_DATA_1000.json').pipe(map((data: any) => {
+      const formattedData: any = { totalCount: 0, results: [] };
```

Comment:
Lots of `any` in this file. Do we actually not know what type of things these are?

Reply:
@AlexOverbeck this is just the test service.